assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -379,3 +382,7 @@ che.infra.openshift.pvc.access_mode=ReadWriteOnce
 # then OpenShift infrastructure will reconfigure installer to use first available from this range
 che.infra.openshift.installer_server_min_port=10000
 che.infra.openshift.installer_server_max_port=20000
+
+# Single port mode wildcard domain host & port. nip.io is used by default
+che.singleport.wildcard_domain.host=NULL
```

Comment:
Make sure you are able to configure it with CLI

Reply:
Added